components/org.wso2.carbon.identity.keyrotation/src/main/java/org.wso2.carbon.identity.keyrotation/config/KeyRotationConfig.java

```diff
@@ -0,0 +1,424 @@
+
+        this.newRegUsername = newRegUsername;
+    }
+
+    /**
+     * Getter for the registry database password.
+     *
+     * @return Registry database password.
+     */
+    public String getNewRegPassword() {
+
+        return new String(Base64.decode(newRegPassword));
+    }
+
+    /**
+     * Setter for the registry database password.
+     *
+     * @param newRegPassword Registry database password.
+     */
+    public void setNewRegPassword(String newRegPassword) {
+
+        this.newRegPassword = newRegPassword;
+    }
+
+    /**
+     * Getter for the enable database migrator property value.
+     *
+     * @return Enable database migrator property value.
+     */
+    public String getEnableDBMigrator() {
+
+        return enableDBMigrator;
+    }
+
+    /**
+     * Setter for the enable database migrator property value.
+     *
+     * @param enableDBMigrator Enable database migrator property value.
+     */
+    public void setEnableDBMigrator(String enableDBMigrator) {
+
+        this.enableDBMigrator = enableDBMigrator;
+    }
+
+    /**
+     * Getter for the enable configuration file migrator property value.
+     *
+     * @return Enable configuration file migrator property value.
+     */
+    public String getEnableConfigMigrator() {
+
+        return enableConfigMigrator;
+    }
+
+    /**
+     * Setter for the enable configuration file migrator property value.
+     *
+     * @param enableConfigMigrator Enable configuration file migrator property value.
+     */
+    public void setEnableConfigMigrator(String enableConfigMigrator) {
+
+        this.enableConfigMigrator = enableConfigMigrator;
+    }
+
+    /**
+     * Getter for the enable syncing migrator property value.
+     *
+     * @return Enable syncing migrator property value.
+     */
+    public String getEnableSyncMigrator() {
+
+        return enableSyncMigrator;
+    }
+
+    /**
+     * Setter for the enable syncing migrator property value.
+     *
+     * @param enableSyncMigrator Enable syncing migrator property value.
+     */
+    public void setEnableSyncMigrator(String enableSyncMigrator) {
+
+        this.enableSyncMigrator = enableSyncMigrator;
+    }
+
+    @Override
+    public String toString() {
+
+        return "\noldSecretKey: " + oldSecretKey + "\nnewSecretKey: " + newSecretKey + "\nnewISHome: " + newISHome +
+                "\noldIdnDBUrl: " + oldIdnDBUrl + "\noldIdnUsername: " + oldIdnUsername + "\noldIdnPassword: " +
+                oldIdnPassword + "\nnewIdnDBUrl: " + newIdnDBUrl + "\nnewIdnUsername: " + newIdnUsername +
+                "\nnewIdnPassword: " + newIdnPassword + "\nnewRegDBUrl: " + newRegDBUrl + "\nnewRegUsername: " +
+                newRegUsername + "\nnewRegPassword: " + newRegPassword + "\nenableDBMigrator: " + enableDBMigrator +
+                "\nenableConfigMigrator: " + enableConfigMigrator + "\nenableSyncMigrator: " + enableSyncMigrator +
+                "\n";
```

Comment:
Where do we use this and do we really need this value?

Reply:
No, this was only done to test the loaded configs, addressed with https://github.com/wso2/identity-tools/pull/13/commits/6b6184cfde572cff05cdeefedb6e9233fd11d0e8